src/tests/execute/cancel_ask_tests.rs

```diff
@@ -338,70 +317,23 @@ mod cancel_ask_tests {
         let mut deps = mock_provenance_dependencies();
         setup_test_base_contract_v3(&mut deps.storage);
 
-        let convertible_marker_json = b"{
-              \"address\": \"tp18vmzryrvwaeykmdtu6cfrz5sau3dhc5c73ms0u\",
-              \"coins\": [
-                {
-                  \"denom\": \"con_base_1\",
-                  \"amount\": \"1000\"
-                }
-              ],
-              \"account_number\": 10,
-              \"sequence\": 0,
-              \"permissions\": [
-                {
-                  \"permissions\": [
-                    \"burn\",
-                    \"delete\",
-                    \"deposit\",
-                    \"admin\",
-                    \"mint\",
-                    \"withdraw\"
-                  ],
-                  \"address\": \"tp18vd8fpwxzck93qlwghaj6arh4p7c5n89x8kskz\"
-                }
-              ],
-              \"status\": \"active\",
-              \"denom\": \"con_base_1\",
-              \"total_supply\": \"1000\",
-              \"marker_type\": \"restricted\",
-              \"supply_fixed\": false
-            }";
-
-        let base_marker_json = b"{
-              \"address\": \"tp18vmzryrvwaeykmdtu6cfrz5sau3dhc5c73ms0u\",
-              \"coins\": [
-                {
-                  \"denom\": \"base_1\",
-                  \"amount\": \"1000\"
-                }
-              ],
-              \"account_number\": 10,
-              \"sequence\": 0,
-              \"permissions\": [
-                {
-                  \"permissions\": [
-                    \"burn\",
-                    \"delete\",
-                    \"deposit\",
-                    \"admin\",
-                    \"mint\",
-                    \"withdraw\"
-                  ],
-                  \"address\": \"tp18vd8fpwxzck93qlwghaj6arh4p7c5n89x8kskz\"
-                }
-              ],
-              \"status\": \"active\",
-              \"denom\": \"base_1\",
-              \"total_supply\": \"1000\",
-              \"marker_type\": \"restricted\",
-              \"supply_fixed\": false
-            }";
-
-        let _base_marker: MarkerAccount = from_binary(&Binary::from(base_marker_json)).unwrap();
-        let _convertible_marker: MarkerAccount =
-            from_binary(&Binary::from(convertible_marker_json)).unwrap();
-        // deps.querier.with_markers(vec![base_marker, convertible_marker]); // TODO: find alternative function
+        // TODO - fix test since mock response returns same result no matter the input
```

Comment:
This is pending a fix from the provwasm team about mocking not considering the inputs of the query

https://discord.com/channels/897185242182484029/1115703566737342474/1128034771436589107

Reply:
The test will still pass as-is though since the code is only checking if the markers are restricted and not taking a look at the other values